one-app-statics/README.md

```diff
@@ -0,0 +1,66 @@
+<h1 align="center">
+  <img src='https://github.com/americanexpress/one-app/raw/master/one-app.png' alt="One App - One Amex" width='50%'/>
+</h1>
+
+# One App Client Assets and CDN
+
+## 🚨 Important 🚨  
+
+This Package is not meant to be installed using npm, it only serves as the CDN for the static client assets for `one-app`,
+if you are looking for `one-app` follow this link: [https://github.com/americanexpress/one-app](https://github.com/americanexpress/one-app)
+
+
+## 👩‍💻 Hiring 👨‍💻
+
+Want to get paid for your contributions to `one-app`?
+> Send your resume to oneamex.careers@aexp.com
+
+## 📖 Table of Contents
+
+* [Features](#-features)
+* [Usage](#-usage)
+* [License](#-license)
+* [Code Of Conduct](#-code-of-conduct)
+* [Contributing](#-contributing)
+
+## ✨ Features
+
+- Improved performance by serving the one app client static assets from a CDN
+- Enhanced security with [Subresource Integrity](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity)
+
+## 🤹‍ Usage with JSDelivr and Docker
```

Comment:
I think this README should be kept to a minimum with just a warning about it not being npm installable and docs on how to use this package could be placed in a `HOW_TO_DEPLOY.md` or similar file and docs on how to use the CDN as a One App user be placed in the running in production recipe I believe @JamesSingleton is working on.

Reply:
@anescobar1991 So to clarify, shall I remove the following sections: 
- Hiring
- Contributing
- License
- Code of Conduct 

I thought they were standard across all the packages that we put out in the wild?

Also regarding the usage and example, do we want just a link to the docs @JamesSingleton is working on, or a brief usage as it is mentioned in lines 38-42 in this repo?

Reply:
@infoxicator,

I actually wanted to sync up with you about working together on that documentation. I have some experience pushing modules to Now and you have experience in pushing it to other sources. It would be awesome to combine our knowledge for the doc!

Reply:
@JamesSingleton Agree, however, this is more relevant to the https://github.com/americanexpress/one-app/blob/master/docs/recipes/Running-In-Production.md guide, I thought you were creating a new guide on how to deploy modules not one app itself? that guide is already too long I think

Reply:
I will be doing a little bit of both. If you think that guide is long, look at the styling one... And we are going to add more UI Libraries

Reply:
This is how the README looks on npm https://www.npmjs.com/package/infoxicator-one-app-statics please take a look and suggest if we still want to make changes to it